### DISCUSSION TITLE
How to close  modal dialog by escape keydown ?

### DISCUSSION THREAD

**Discussion by sergeynilov** [alpinejs/alpine#1215] [1. Help]
Hello,
With AlpineJs 2.8.2 I make modal dialog
and I want to close it by Cancel key, but setting keydown event for cancel button(it is inside of component) :

```
     <div class="flex justify-end pt-2 modal-footer ">
         <button type="button" class="editor_form_cancel_btn" @click="isOpenSystemInfoDialogModalOpened = false" @keydown.escape="isOpenSystemInfoDialogModalOpened = false" >  Cancel
         </button>    
      </div>

```
I suppose button must acces cancel key event, but failed.

Please look at codepen :
https://codepen.io/sergeynilov/pen/vYyPrrE?editors=1010

Thanks!

**Comment by ryangjchandler** [comment]
Here you go @sergeynilov - you were missing the `.window` modifier: https://codepen.io/ryangjchandler/pen/abpBoXw?editors=1010
